src/foam/nanos/dig/SugarWebAgent.java

```diff
@@ -6,29 +6,37 @@
 
 package foam.nanos.dig;
 
-import foam.dao.DAO;
+import java.io.PrintWriter;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.lang.reflect.Parameter;
+import java.nio.CharBuffer;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.gson.Gson;
```

Comment:
Why are we using this library?

Reply:
@kgrgreer .. how do we parse JSON into a map of key value pairs?

Reply:
I switch it to foam MapParser